db/migrate/20151229184253_create_token_auth_synchronizable_resources.rb

```diff
@@ -13,3 +13,13 @@ def change
     end
   end
 end
+
+if ActiveRecord::Migration.respond_to? :[]
+  class CreateTokenAuthSynchronizableResources < ActiveRecord::Migration[4.2]
```

Comment:
```ruby
ActiveRecord::Migration[4.2]
```
vs.
```ruby
ActiveRecord::Migration
```

In Rails 5, you have to specify the Rails version for which the migration was written. In this case, it was `4.2`.